From: Matthias Bach <marix@marix.org>
Date: Wed, 3 Jan 2024 01:10:05 +0100
Subject: [PATCH] Fix compatibility with Python 3.12
References: https://github.com/dschep/ntfy/pull/271
Upstream: submitted

Unittest's assertEquals has finally been dropped in Python 3.12.
---
 tests/test_cli.py | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/tests/test_cli.py b/tests/test_cli.py
index c1341f3..4a87911 100644
--- a/tests/test_cli.py
+++ b/tests/test_cli.py
@@ -174,7 +174,7 @@ class TestMain(TestCase):
     @patch('ntfy.backends.default.notify')
     def test_args(self, mock_notify):
         mock_notify.return_value = None
-        self.assertEquals(0,
+        self.assertEqual(0,
                           ntfy_main([
                               '-o', 'foo', 'bar', '-b', 'default', '-t',
                               'TITLE', 'send', 'test'
-- 
2.35.3

